blueocean-rest-impl/src/main/java/io/jenkins/blueocean/service/embedded/rest/UserImpl.java

```diff
@@ -36,15 +44,20 @@
 
     protected final User user;
 
+    private final BlueOrganization organization;
+    private final AccessControlled organizationBase;
+
     private final Reachable parent;
-    public UserImpl(User user, Reachable parent) {
+
+    public UserImpl(@Nonnull BlueOrganization organization, @Nonnull User user, Reachable parent) {
```

Comment:
Well I'm not sure you will have the organization every where you want to check for the permissions. I was asked to follow the same path followed here https://github.com/jenkinsci/blueocean-plugin/pull/1278

Also I think we want to treat the users as belonging to an organization when we have multiple of them being able to only retrieve the ones with permissions in a given organizations in which case this could make more sense.